### DISCUSSION TITLE
Phpmyadmin route keeps returning 404 error.

### DISCUSSION THREAD

**Discussion by Charaoh** [leokhoa/laragon#652] [Q&A] [answered]
Everytime I go to localhost/phpmyadmin I get directed to this.

![image](https://github.com/leokhoa/laragon/assets/12887020/4d248ae2-0b46-4873-a7b9-6935061cb3cc)

I have unstalled and reinstalled the the program, removed phpmyadmin and reinstalled another version. Rebooted, yet for some reason I cannot access it. Ive been using Heidi, however, anyway to get this resolved?

**Comment by Manto-X** [comment, marked as answer]
Do add phpmyadmin firstly by go right click on Laragon tray icon, then select: Tool > Quick add > *phpmyadmin

![image](https://github.com/leokhoa/laragon/assets/32100754/6f01fa09-44ce-4c41-bf8f-d733d763aca7)

**Reply by f6de** [reply]
ty man u a life saver

**Reply by WUIX69** [reply]
Why mine shows this, i added both phpMyadmin and 6.0 snapshot on quickadd

![{C9EBA819-FF51-4A77-AC70-50251D8D40DB}](https://github.com/user-attachments/assets/c16b6966-f6b8-4b13-b811-fca040a98259)
